LinkedList cpp/linked_list.cpp

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <iostream>
+#include <cassert>
+#include <sstream>
+using namespace std;
+
+class Node {
+public:
+    int data;
+    Node* next;
+
+    Node(int data) {
+        this->data = data;
+        this->next = nullptr;
+    }
+};
+
+class LinkedList {
+public:
+    Node* head;
+
+    LinkedList() {
+        head = nullptr;
+    }
+
+    void insert(int data) {
+        Node* new_node = new Node(data);
+        if (head == nullptr) {
+            head = new_node;
+        } else {
+            Node* current = head;
+            while (current->next != nullptr){
+                current = current->next;
+            }
+            current->next = new_node;
+        }
+    }
+    void print() {
+        Node* current = head;
+        while (current != nullptr) {
+            cout<<current->data;
+            cout<<" --> ";
```

Comment:
```suggestion
            cout << current->data;
            cout << " --> ";
```

Add some spaces around << operator. Again code formatter would take care of this.